manifest/common.go

```diff
@@ -63,7 +64,7 @@ func compressionVariantMIMEType(variantTable []compressionMIMETypeSet, mimeType
 			if mt == mimeType { // Found the variant
 				name := mtsUncompressed
 				if algorithm != nil {
-					name = algorithm.Name()
+					name = strings.Split(algorithm.Name(), ":")[0]
```

Comment:
Is this really a _generic_ mechanism for MIME types? If not, I’d prefer 1) expressing this as table-driven data, or 2) an explicit exception for that particular pair of strings, not introducing a generic code behavior with unspecified semantics.

(I know, it is far too early to discuss such details.)